client/me/help/help-contact-form/index.jsx

```diff
@@ -94,6 +95,18 @@ module.exports = React.createClass( {
 		this.setState( { siteSlug } );
 	},
 
+	trackClickStats: function( selectionName, option ) {
```

Comment:
Here we could be more specific - since the `trackClickStats` function doesn't need the whole `option` object, we could just accept the `selected_option` directly:

```js
trackClickStats: function( selectionName, select_option ) {
   // ...
}
```